core/src/main/groovy/noe/common/utils/Cmd.groovy

```diff
@@ -495,7 +495,7 @@ public class Cmd {
    * returns kill process exit code. If PID is not defined on non-windows platform, -1 is returned
    * TODO: properly implement it for Unix/Linux systems as pkill -P kills only processes with given parent PID, it doesn't go recursively killing from bottom to top
    */
-  static int killTree(Integer pid, winappname) {
+  static Long killTree(Long pid, winappname) {
```

Comment:
@jstourac please can you verify the return type as looking at the code it could have remained an int, if I'm not mistaken.

Reply:
@PaulLodge, you're right, thanks for the catch - I've just updated fix. I'll also start internal test jobs again, just to be sure